### PR TITLE
Jetson nano

### PR DESCRIPTION
**Pull request by bestaps**
I feel the Nano scripts are useful for beginners who are using Jetson Nano.

### CODE REVIEW

**Comment by magicrub** [comment]
Ping @jmachuca77

File: Nvidia_Nano/Ubuntu/bashrc-final [hunk 1]
```diff
@@ -126,4 +126,4 @@ fi
 
 export LS_COLORS="di=96:ex=92:"
 
-export PATH=$PATH:/home/apsync/start_mavproxy
+export PATH=$PATH:/home/jaime/start_mavproxy
```
Comment (davidbuzz) [inline Nvidia_Nano/Ubuntu/bashrc-final:129]: i don't think we want 'jaime' in the repo.

Comment (jmachuca77) [reply Nvidia_Nano/Ubuntu/bashrc-final:129]: Lol.. no.

**Comment by jmachuca77** [comment]
I didn't realize these changes weee being pushed to the PR... I'll fix them up.

**Comment by rmackay9** [comment]
@jmachuca77,

No big deal of course but any chance you could improve the commit title?  Maybe make it start with "Nvidia_Nano:" or something like that.  Just a bit more specificity so we can see what each commit affects

**Comment by jmachuca77** [comment]
Sorry this pull request is very outdated the commit was not supposed to go into a pr. Closing. By the way I will be ditching these scripts in favor of a docker container

**Pull request closed without merging**